tests/acceptance/tasks/ShopAdmin/Category/CreateLandingPage.ts

```diff
@@ -0,0 +1,47 @@
+import { test as base } from '@playwright/test';
+import type { FixtureTypes, Task } from '@fixtures/AcceptanceTest';
+
+export const CreateLandingPage = base.extend<{ CreateLandingPage: Task }, FixtureTypes>({
+    CreateLandingPage: async ({ ShopAdmin, AdminCategories, AdminLandingPageCreate, AdminLandingPageDetail, TestDataService }, use ) => {
+
+        const task = (layoutName: string, landingPageData, status: boolean) => {
+            return async function CreateLandingPage() {
+                await AdminCategories.landingPageHeadline.click();
+                await AdminCategories.addLandingPageButton.click();
+
+                await ShopAdmin.expects(AdminLandingPageDetail.saveLandingPageButton).toBeVisible();
+                await ShopAdmin.expects(AdminLandingPageDetail.saveLandingPageButton).toContainText('Save');
+
+                //Fill details and save
+                await AdminLandingPageCreate.nameInput.fill(landingPageData.name);
+                await AdminLandingPageCreate.landingPageStatus.setChecked(landingPageData.status);
+                await AdminLandingPageCreate.salesChannelSelectionList.click();
+                await AdminLandingPageCreate.filtersResultPopoverItemList.filter({ hasText: landingPageData.salesChannel }).click();
+                await AdminLandingPageCreate.seoUrlInput.fill(landingPageData.seoUrl);
+
+                if (layoutName) {
+                    await AdminLandingPageCreate.layoutTab.click();
+                    // Verify empty layout state
+                    await ShopAdmin.expects(AdminLandingPageCreate.layoutEmptyState).toBeVisible();
+                    await ShopAdmin.expects(AdminLandingPageCreate.createNewLayoutButton).toBeVisible();
+                    // Select existing layout
+                    await AdminLandingPageCreate.assignLayoutButton.click();
+                    await AdminLandingPageCreate.loadingSpinner.waitFor({ state: 'hidden' });
+                    await AdminLandingPageCreate.searchLayoutInput.dblclick();
+                    // Search input need to delay press more than 300ms to mimic user typing in order to activate search action
+                    await AdminLandingPageCreate.searchLayoutInput.pressSequentially(layoutName.split(' ')[1].substring(0,5), {delay: 500});
+                    await AdminLandingPageCreate.layoutItems.first().waitFor({ state: 'visible' });
+                    await AdminLandingPageCreate.layoutItems.first().click();
```

Comment:
I would shorten that way with:

`await AdminLandingPageCreate.getByTitle(layoutName).click();`

And to make sure your layout is everytime on first page (mostly the time) you can use the prefix `000_layoutPageName_ ....`

What do you think about it?

Reply:
I think your is clever and is adding more stability as well as debugging capability. 
Keeping the search to me was to prevent concurrency and sorting order change failure, but not 100% sure the benefit is worth it. So I'm adapting this change.